planner/core/logical_plan_test.go

```diff
@@ -19,6 +19,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/pingcap/parser/ast"
+
```

Comment:
please remove this empty line.

Reply:
fixed. I assumed all empty lines should be removed right?

Reply:
No. We split the import packages to two parts. The first part is the golang standard libs, the second part is the third-part libs. Between these two parts, there should be an empty line.

Reply:
Aha. Got it. Thanks